scoring/score.py

```diff
@@ -1,9 +1,57 @@
+DATA = {
+    # Location -> (Owner, Points if owner matches)
+    'arena':              (None, 0),
+    'docking_area':       (None, 1),
+    'zone_0_raised_area': (0,    3),
+    'zone_1_raised_area': (1,    3),
+}
+
+
 class Scorer:
+    # Assumption: we will trust the *ordering* of the entries in the data we
+    # are given and *ignore* the actual 'time' values. We defer handling of
+    # equialent-time operations to the token controller.
+
     def __init__(self, teams_data, arena_data):
-        pass
+        self._zone_to_tla = {
+            info['zone']: tla
+            for tla, info in teams_data.items()
+        }
+
+        # Token Claims look like this:
+        # {
+        #     'zone': 0 | 1,
+        #     'token_index': int,
+        #     'location': 'arena' | 'docking_area' | 'zone_0_raised_area' | 'zone_1_raised_area',
+        #     'time': float,
+        # }
```

Comment:
Worth capturing into a dataclass/namedtuple?

Reply:
We don't normally. There's not automated checking in this repo, so the typing benefits are minimal.
(I do have a plan for getting generalised checking, but we're not there yet)